### PR TITLE
Intro functions and cursor

### PR DESCRIPTION
**Pull request by studioant**
Additions:
Thoth Tech Company Intro with sound
SplashKit Productions loading screen
Moved get_from_git into Splashkit loading screen
Helper.get_from_git returns bool
Add music to menu screen
Changed Arcade Machine title font
Added creators text
Added SplashKit cursor  
Button.action added actions and cout
Button.action returns a string
Selector.check_key_input returns string

### CODE REVIEW

**Review by sarahgos — APPROVED** [review]
This all looks great to me Anthony, have just run it and I love the animations in the intro screen and the music choice is perfect. I also think the SplashKit cursor is a good addition, bringing the theme together. Fantastic work Anthony!

File: helper.h [hunk 1]
```diff
@@ -175,4 +175,25 @@ class Helper{
                 refresh_screen(60);
             }
         }
+
+        void play_intro()
```
Comment (Delcari) [inline helper.h:179]: Definitely need to comment this, from my understanding you are covering the splash-screen with a white rectangle and fading it out.

**Review by Delcari — APPROVED** [review]
The intro, loading screen and music are great. 
Please review aforementioned comment, lovely work Anthony!

File: helper.h (hunk 1 above)
Comment (studioant) [reply helper.h:183]: Cheers Riley, Yes that's right, I have commented the function as suggested.

**Pull request merged**